include/realizations/catchment/Bmi_Multi_Formulation.hpp

```diff
@@ -275,11 +275,30 @@ namespace realization {
          * @return The inclusive beginning of the period of time over which this instance can provide this data.
          */
         time_t get_forcing_output_time_begin(const std::string &forcing_name) override {
+            std::string var_name = forcing_name;
+            if(var_name == "*" || var_name == ""){
+                // when unspecified, assume all data is available for the same range.
+                // Find one that successfully returns...
+                for(std::map<std::string,std::shared_ptr<forcing::ForcingProvider>>::iterator iter = availableData.begin(); iter != availableData.end(); ++iter)
+                {
+                    var_name = iter->first;
+                    std::cerr<<"Trying var_name "<<var_name<<std::endl;
```

Comment:
Also, do we want to remove these prints?

Reply:
Yes, thought I did, sorry 🤔